src/webgl/p5.RendererGL.js

```diff
@@ -446,6 +448,17 @@ p5.RendererGL = class RendererGL extends p5.Renderer {
     this.spotLightAngle = [];
     this.spotLightConc = [];
 
+    // repetition
+    this.textures = new Map();
```

Comment:
I think we don't need to declare this any more now that we have `blurryTextures` below

Reply:
Sure, I will remove this. I was going to ask you about this, that's the reason I marked it as repetition.